### PR TITLE
Added the Symfony default ide file link web view

### PR DESCRIPTION
**Pull request by jeremyFreeAgent**
This add the feature added by https://github.com/symfony/symfony/pull/19973

### CODE REVIEW

**Comment by SpacePossum** [comment]
We only know if this works if your SF PR is merged, as the class `FileLinkFormatter` doesn't exists yet :)
(maybe add a utest for this change?)

**Comment by fabpot** [comment]
I supposed @jeremyFreeAgent tested it manually, right?

**Comment by jeremyFreeAgent** [comment]
Yes I needed to test it manually since the dependencies are not merged yet. If the `FileLinkFormatter` is not here, it fallbacks on the previous one.

**Comment by jeremyFreeAgent** [comment]
I've pushed the changes to reflect the ones made in the Symfony PR.

File: WebProfilerServiceProvider.php [hunk 1]
```diff
@@ -69,6 +70,19 @@ public function register(Container $app)
...
+        $rootDir = end($rootDir)['file'];
+        $rootDir = explode(DIRECTORY_SEPARATOR, realpath($rootDir) ?: $rootDir);
+        $providerDir = explode(DIRECTORY_SEPARATOR, __DIR__);
+        for ($i = 0; isset($rootDir[$i], $providerDir[$i]); ++$i) {
+            if ($rootDir[$i] !== $providerDir[$i]) {
+                break;
+            }
+            $baseDir[] = $rootDir[$i];
+        }
+        $baseDir = implode(DIRECTORY_SEPARATOR, $baseDir);
```
Comment (fabpot) [inline WebProfilerServiceProvider.php]: Can you move this into a private method?

Comment (jeremyFreeAgent) [inline WebProfilerServiceProvider.php]: Done!

**Comment by fabpot** [comment]
Thank you @jeremyFreeAgent.

**Pull request merged**